lib/diagnostic.js

```diff
@@ -4,23 +4,39 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-const min = function min() {
-
+const min = function min(set) {
+  let smallest = 0;
+  for (let i = 0; i < set.length; i++) {
+    if (set[i]<smallest) {
+      smallest = number[i]
+    }
+    console.log(smallest)
+  }
 };
 
-// Question 2
+
+
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-const sum = function sum() {
-
+const sum = function sum(numbers) {
+  let stuff = 0;
+  for (let i = 0; i < numbers.length; i++) {
+    stuff += numbers[i];
+  }
+  return stuff;
 };
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 const newArray = function newArray(length, defaultValue) {
-
+  if (Array.isArray(lenth)) {
+    let prod = 1;
+    for (let i = 0; i < length.length; i++) {
+      prod = numbers[i];
+    }
+    return prod;
```

Comment:
watch for syntax errors here. you didn't close the if statement and wrote lenth in Array.isArray isntead of length